src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -158,6 +158,11 @@ public void setPerson(Person target, Person editedPerson) {
             throw new AssertionError("This method should not be called.");
         }
 
+        @Override
+        public Boolean tagExists(Person target, Set<Tag> tags) {
```

Comment:
I may be wrong but I think you are retrning the Boolean wrapper, instead of the primitive Boolean. Not sure if this is intentional?

Reply:
Boolean wrapper should be fine as well i believe?

Reply:
Yep, logic wise, it should be the same, but there may be extra overhead